File: frontend/src/app/dijagnoza/dijagnozaGetAllResponse.ts
export interface DijagnozaGetAllResponse{
  dijagnoze:DijagnozaGetAllResponseDijagnoza[]
}

export interface DijagnozaGetAllResponseDijagnoza
{
  dijagnozaId: number
  nazivBolesti: string
  opis: string
  datumDijagnoze: string
  zaposlenikId: number
  korisnikDomaID: number
}

File: DomZaStaraLicaApi/Endpoints/Dijagnoza/GetAll/DijagnozaGetAllResponse.cs
namespace DomZaStaraLicaApi.Endpoints.Dijagnoza.GetAll
{
    public class DijagnozaGetAllResponse
    {
        public List<DijagnozaGetAllResponseDijagnoza> Dijagnoze {  get; set; }
    }

    public class DijagnozaGetAllResponseDijagnoza
    {
        public int dijagnozaId { get; set; }
        public string nazivBolesti { get; set; }
        public string opis { get; set; }
        public DateTime datumDijagnoze { get; set; }
        public int ZaposlenikId { get; set; }
        public int KorisnikDomaID { get; set; }
    }
}
